e2etests/orchestrator_scan_test.go

```diff
@@ -20,6 +20,34 @@ var (
 	patchRegex = regexp.MustCompile(`^[0-9]+\.[0-9]+\.([0-9]+)$`)
 )
 
+func TestGRPCGetIstioVulnerabilities(t *testing.T) {
```

Comment:
Just want to make sure `GetIstioVulnerabilitiesRequest()` and `GetIstioVulnerabilities() ` are accepting requests and returning correct responses.

Reply:
I think it may be more worthwhile to give an Istio version and ensure we find specific vulnerabilities. For example, we know Istio verison 1.13.6 is affected by ISTIO-SECURITY-2022-007, which has a specific description and CVSS and is fixed by 1.13.9